react-native-community/cli
# feat: extract android applicationId

Summary:
---------

Use `applicationId` specified in `defaultConfig` of `build.grade` file. 

Fixes #1884 

Test Plan:
----------

1. Clone the fork and checkout to current branch
2. Follow [Contributing guide](https://github.com/react-native-community/cli/blob/main/CONTRIBUTING.md) (link CLI)
3. Change default `applicationId` inside `defaultConfig` of `build.gradle` file, eg.
```diff
 namespace "com.foo"
    defaultConfig {
+        applicationId "com.bar"
-        applicationId "com.foo"
```
4. Run the app

----------

TODO's / things to consider 🤔 :
- [ ] we do not always have `appld` present at `defaultConfig` object in `build.gradle` file. Should we default then to namespace?
- [ ] users can have different applicationId's specified for build variants / flavors. Do we want to cover those cases as well?
